utils/docker-ce/files/dockerd.init

```diff
@@ -135,6 +135,7 @@ process_config() {
 
 	config_get data_root globals data_root "/opt/docker/"
 	config_get log_level globals log_level "warn"
+	config_get_bool iptables globals iptables "1"
```

Comment:
Since we don't do all config options we should probably add this to the config file for documentation.
I'd prefer we had `'true'` in the config file instead of a number so it matches the JSON closer.

Reply:
> Since we don't do all config options we should probably add this to the config file for documentation.

I added the option to the uci template configuration as you suggested.

> I'd prefer we had 'true' in the config file instead of a number so it matches the JSON closer.

I understand but in the uci I think it is better to use 0 or 1 for boolean.
The data become valid json bythe function json_add_boolean.
This function converts the 0 and 1 to a valid boolean.
If we take true or false, then i have to adjust the if state as well.

I think and my experience with openwrt is, that it is rather normal to work with 0 or 1 at bool values.

Reply:
From the scripting perspective [It looks like we'd still get a 1/0](https://github.com/openwrt/openwrt/blob/master/package/base-files/files/lib/functions.sh#L122), so I thought it would be fine. That said, it was just a personal preference so whatever you think is best works for me.